src/index.ts

```diff
@@ -3,8 +3,6 @@ export { default as Cell } from './Cell';
 export { default as Row } from './Row';
 export { default as HeaderCell } from './HeaderCell';
 export * from './Columns';
-export * from './Cell/cellContentRenderers';
-export { default as CellContent } from './Cell/CellContent';
```

Comment:
https://github.com/adazzle/react-data-grid/pull/1831
@qili26 Why did we need this export for again?

Reply:
@nstepien  @amanmahajan7  Just DM you guys.